.github/workflows/build_and_run_esp_usb_test_apps.yml

```diff
@@ -0,0 +1,80 @@
+name: ESP-USB Test Apps
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Every day at midnight
```

Comment:
We added cron job, to run every day, to test the docker fix more frequently. 
I think we can leave it here for some time. Just to keep in mind that we normally don't have to run this job this frequently.

Reply:
I will check it and probably disable it completely. I think that I want it to run only by the pull-request.

Reply:
Removed cron completely.